starter-code/css/main.css

```diff
@@ -6,5 +6,99 @@ light blue: #1c5380
 
 body {
   font: 200 16px/1.5 "Helvetica Neue", Helvetica, Arial, sans-serif;
-  margin: 0;
+  margin: 0 auto;
+  width: 100%;
+  background-image: url("../images/bgtile.png");
+  background-repeat: repeat-x;
+  /* background-size: cover; */
+}
+
+.wrapper{
+  width: 1000px;
+  margin: 125px auto;
+}
+
+.left-column{
+  
+}
+
+.right-column{
+ 
+}
+
+
+
+.left-float{
+  float: left;
```

Comment:
you can pass this to the class that will be using it instead of creating another class for it